src/services/notification/index.ts

```diff
@@ -0,0 +1,61 @@
+import config from 'config'
+import { Service } from 'feathers-sequelize'
+
+import { Application, CachedService, ServiceAddresses } from '../../definitions'
+import notificationHooks from './notification.hooks'
+import notificationChannels from './notification.channels'
+import { loggingFactory } from '../../logger'
+import { waitForReadyApp } from '../../utils'
+import NotificationModel from './notification.model'
+import {
+  messageHandler,
+  Comms
+} from '../../communication'
+import { sleep } from '../../../test/utils'
+
+export class NotificationService extends Service {
+  emit?: Function
+}
+
+const NOTIFICATION = 'notification'
+
+const logger = loggingFactory(NOTIFICATION)
+
+const notification: CachedService = {
+  async initialize (app: Application): Promise<{ stop: () => void }> {
+    if (!config.get<boolean>(`${NOTIFICATION}.enabled`)) {
+      logger.info('Notification service: disabled')
+      return { stop: () => undefined }
+    }
+    logger.info('Notification service: enabled')
+
+    await waitForReadyApp(app)
+
+    // Initialize Notification service
+    app.use(ServiceAddresses.NOTIFICATION, new NotificationService({ Model: NotificationModel }))
+    const notificationService = app.service(ServiceAddresses.NOTIFICATION)
+    notificationService.hooks(notificationHooks)
+    app.configure(notificationChannels)
+
+    // Init comms
+    const comms = app.get('comms') as Comms
+    comms.messageHandler = messageHandler(notificationService)
+
+    return {
+      stop: () => undefined
+    }
```

Comment:
maybe you could DRY this is a bit with a const.

Reply:
you mean create a const for stop function like: `const emptyStop = () => undefined`?

Reply:
yea it's a small point. it's just that it is used twice in the fine so might as well. but it's up to you of course

Reply:
Done, thanks